### PR TITLE
Add 'Copy Node Path' action to right mouse menu

### PR DESCRIPTION
**Pull request by Geequlim**
This should make external text editor happy :P

### CODE REVIEW

File: tools/editor/scene_tree_dock.cpp [hunk 1]
```diff
@@ -1807,11 +1820,11 @@ void SceneTreeDock::_tree_rmb(const Vector2& p_menu_pos) {
 		menu->add_separator();
 		menu->add_icon_shortcut(get_icon("Blend","EditorIcons"),ED_GET_SHORTCUT("scene_tree/merge_from_scene"), TOOL_MERGE_FROM_SCENE);
 		menu->add_icon_shortcut(get_icon("Save","EditorIcons"),ED_GET_SHORTCUT("scene_tree/save_branch_as_scene"), TOOL_NEW_SCENE_FROM);
+		menu->add_separator();
+		menu->add_icon_shortcut(get_icon("Duplicate","EditorIcons"), ED_GET_SHORTCUT("scene_tree/copy_node_path"), TOOL_COPY_NODE_PATH);
```
Comment (akien-mga) [inline tools/editor/scene_tree_dock.cpp]: I guess this would need its own clipboard icon :)

**Review by akien-mga — APPROVED** [review]

File: tools/editor/scene_tree_dock.cpp (hunk 1 above)
Comment (Geequlim) [inline tools/editor/scene_tree_dock.cpp]: I agree with you.
But can make a beautiful new icon for it

Comment (akien-mga) [inline tools/editor/scene_tree_dock.cpp]: Stealthy CC of @djrm :)
I wonder if we don't have already a fitting icon for the action of copying.

Comment (djrm) [inline tools/editor/scene_tree_dock.cpp]: dont worry, i will check that out and create an icon for it.

**Pull request merged**